plugins/RSS/plugin.py

```diff
@@ -719,17 +724,22 @@ def rss(self, irc, msg, args, url, n):
                 s += str(feed.last_exception)
             irc.error(s)
             return
-        n = n or self.registryValue('defaultNumberOfHeadlines', channel, irc.network)
         entries = list(filter(lambda e:self.should_send_entry(irc.network, channel, e),
                               feed.entries))
-        entries = sort_feed_items(entries, 'newestFirst')
+        upper_sort = 'newestFirst'
+        if [y for x,y in optlist if x == 'random']:
+            upper_sort = 'random'
+        entries = sort_feed_items(entries, upper_sort)
```

Comment:
```suggestion
        order = 'newestFirst'
        if 'random' in (x for x,y in optlist):
            order = 'random'
        entries = sort_feed_items(entries, order)
```

looks more readable to me